dashboard/app/views/levels/editors/_cs_in_a.html.haml

```diff
@@ -1,26 +1,35 @@
 //CS in A stuff
-.field
-  = render partial: 'levels/editors/checkboxes', locals: {f: f, field_name: :use_contract_editor, description: "Use MSM contract editor"}
-.field
-  = render partial: 'levels/editors/checkboxes', locals: {f: f, field_name: :default_num_example_blocks, description: "Default # of example blocks in contract editor"}
-.field
-  = render partial: 'levels/editors/checkboxes', locals: {f: f, field_name: :disable_examples, description: "Hide examples section in contract editor"}
-.field
-  = render partial: 'levels/editors/checkboxes', locals: {f: f, field_name: :examples_required, description: "Each function must have at least 1 example, and examples will be checked when hitting run"}
-%table.contract-editor-config
-  %tr
-    %th Section
-    %th Highlight
-    %th Collapse
-  -%w(contract examples definition).each_with_index do |section, i|
-    %tr
-      %td #{i + 1}. #{section.titleize}
-      %td= boolean_check_box f, "#{section}_highlight".to_sym
-      %td= boolean_check_box f, "#{section}_collapse".to_sym
+%legend.control-legend{data: {toggle: "collapse", target: "#csInA"}}
+  CS in A Options
 
--if @level.respond_to? :input_output_table
+#csInA.collapse
```

Comment:
naming nit: hyphenate CSS ID name -- `cs-in-a`

Reply:
I think I probably did this for all the names - so I'll go through and update in the last PR